third_party/txt/src/txt/font_collection.cc

```diff
@@ -250,6 +250,28 @@ void FontCollection::SortSkTypefaces(
         int a_delta = std::abs(a_style.width() - SkFontStyle::kNormal_Width);
         int b_delta = std::abs(b_style.width() - SkFontStyle::kNormal_Width);
 
+        {
+          // A workaround to prevent emoji fonts being selected for normal text
+          // when normal and emojis are mixed at same font family.
+
+          bool a_isContainEmoji = false;
+          bool b_isContainEmoji = false;
```

Comment:
From what I guess from the code the correct naming would be
```suggestion
          bool a_isEmojiFont = false;
          bool b_isEmojiFont = false;
```

Otherwise it could also be
```suggestion
          bool a_containsEmoji = false;
          bool b_containsEmoji = false;
```